scalafmt-tests/src/test/resources/scala3/GivenPatterns.stat

```diff
@@ -0,0 +1,27 @@
+#
```

Comment:
this one is not related to a code change, right?

Reply:
right, for test purposes only